### DISCUSSION TITLE
Preview 12 ETA?

### DISCUSSION THREAD

**Discussion by leastprivilege** [dotnet/yarp#1089] [Q&A] [answered]
With the recent changes, I wanted to wait for the next official preview before we ship our BFF library. Do you have a date?

thanks!

**Comment by Tratcher** [comment, marked as answer]
Just wrapping up some loose ends like https://github.com/microsoft/reverse-proxy/pull/1087, we want to get most of the API breaks handled in this preview. We should be ready in the next week or so.

**Reply by leastprivilege** [reply]
thanks!

**Reply by Tratcher** [reply]
https://github.com/microsoft/reverse-proxy/releases/tag/v1.0.0-preview12

**Reply by leastprivilege** [reply]
Saw it! thanks!
